### PR TITLE
Sound stops when user clicks File->Close

### LINKED ISSUE #65: File->Close only closes the window, music still runs

### PR DESCRIPTION
**Pull request by joshhting**
Fixed #65

### CODE REVIEW

File: src/main/java/io/github/mountainrange/mule/MULE.java [hunk 1]
```diff
@@ -116,6 +117,7 @@ public void setFullscreen(Boolean b) {
 	}
 
 	public void close() {
+		Config.getInstance().soundManager.stopAllPlaylists();
```
Comment (jgkamat) [inline src/main/java/io/github/mountainrange/mule/MULE.java]: Can we change this to 

```
Platform.exit();
System.exit(0);
```

if possible, if that dosent work, go ahead and merge this.

**Comment by joshhting** [comment]
Alright I changed it. I originally chose not to do it that way because I assumed that `primaryStage.setOnCloseRequest()` was already being called implicitly as a side effect of calling `primaryStage.close()`. Guess I assumed wrong

**Comment by jgkamat** [comment]
It might be actually called, I'm just adding it to be safe. System.exit is what actually closes it.

**Comment by jgkamat** [comment]
I'm merging now, +1

**Pull request merged**